src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,94 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+import { useAppDispatch, useAppSelector } from '../../app/hooks';
+import { actions as actionsCurrent } from '../../features/currentTodo';
+
+const errorText = 'User not found';
 
 export const TodoModal: React.FC = () => {
+  const dispatch = useAppDispatch();
+  const selectedTodo = useAppSelector(state => state.currentTodo);
+  const [user, setUser] = useState<User | null>(null);
+  const [errorMessage, setErrorMessage] = useState<string>('User not found');
```

Comment:
The initial state of errorMessage is set to 'User not found', but it should be empty or a more generic message, since you can't know the error before it happens.